src/screens/block/block.vue

```diff
@@ -113,10 +116,27 @@ export default {
       } else if (blockHashRegex.test(this.blockId)) {
         this.$store.dispatch('loadBlock', { hash: this.blockId })
       }
+    },
+    keyWatcher () {
+      let self = this
+      window.addEventListener('keyup', function (ev) {
+        self.blockNavigation(ev)
+      })
+    },
+    blockNavigation (ev) {
+      if (this.$route.name === 'Block') {
+        if (ev.keyCode === 39 && this.height > this.block.height) {
+          this.blockId++
```

Comment:
I get loads of warnings to not mutate a prop directly. I think you should introduce a data field (e.g. displayBlockId or currentBlockId) which on init gets the value of blockId and then increment and decrement this value. Also use this data field in the rest of the code instead of blockId.

Reply:
aaalso in the getBlock() method I saw that its possible to load a block by block hash. Then blockId is a string containing a hash `http://localhost:8080/#/block/bh$2NHAwCnvhqwnJ7zUm5PszBBAM5YJQBaMQfZipQ9mF2g5dxADxb` which breaks navigation